storage/gcloud/aio/storage/storage.py

```diff
@@ -172,6 +173,22 @@ async def _headers(self) -> Dict[str, str]:
             'Authorization': f'Bearer {token}',
         }
 
+    async def get_buckets(
+        self, project: str, timeout: int = DEFAULT_TIMEOUT,
+    ) -> Set[Bucket]:
+        url = ('https://storage.googleapis.com/'
+               f'storage/v1/b?project={project}')
```

Comment:
We build the API root in the contructor of this class so we can eg. support the GCS emulator and such. Please make use of the pre-built variable:
```suggestion
        url = f'{self._api_root_read}?project={project}'
```

Reply:
Will do